app/src/main/java/c4stor/com/feheroes/activities/ivcheck/IVCheckActivity.java

```diff
@@ -145,10 +152,10 @@ public void onNothingSelected(AdapterView<?> parent) {
             }
         });
 
-        populateSpinner(R.id.spinner_stars, R.array.stars_array);
+        populateSpinner();
 
         adAdBanner();
-//        disableAdBanner();
+        disableAdBanner();
```

Comment:
Please don't commit both adAdBanner and disableAdBanner uncommented, I can get banned from the play store for displaying invisible ads ^^